helm_deploy/hmpps-prisoner-to-nomis-update/values.yaml

```diff
@@ -4,6 +4,14 @@ generic-service:
 
   replicaCount: 4
 
+  resources:
+    requests:
+      cpu: 10m
+      memory: 640Mi
+    limits:
+      cpu: 2000m
+      memory: 2048Mi
```

Comment:
This overrides the limit range defined in the Cloud Platform terraform, which is currently set to a max of 1024Mi. As by default the heap gets 1/4 of the max, this explains why the the heap is always >90% full for this app.

We could explicitly set the size of all JVM memory components including the heap as explained in [this article](https://cloud.theodo.com/en/blog/jvm-oom). But 1/4 seems fairly sensible to me and it's the default for a reason.